ui.pyw

```diff
@@ -185,7 +185,16 @@ def main():
     root = Tk()
     root.geometry("700x390+200+200")
     root.minsize(700, 390)
-    root.iconbitmap(join("images", 'daffy.ico'))
+    
+    #set the root icon, for some reason Linux acts a bit special here
+    from sys import platform as _platform
+    if _platform == "linux" or _platform == "linux2":
+        daffy_image=join("images", 'daffy.png')
+        img = PhotoImage(file=daffy_image)
```

Comment:
the generic name image points out how unneeded this var is, can just inline the call to the next line